extensions/amp-story/1.0/progress-bar.js

```diff
@@ -155,12 +157,17 @@ export class ProgressBar {
             this.clear_();
           }
 
+          const storyAdSegmentBranch = getExperimentBranch(
+            this.win_,
+            StoryAdSegmentExp.ID
+          );
           /** @type {!Array} */ (pageIds).forEach((id) => {
             if (
               // Do not show progress bar for the ad page (control group).
               // Show progress bar for the ad page (experiment group).
               (!this.isAdSegment_(id) ||
-                isExperimentOn(this.win_, 'story-ad-auto-advance')) &&
+                (storyAdSegmentBranch &&
```

Comment:
If `storyAdSegmentBranch` is null, you don't need to check `x && x == CONTROL`, you can just check `x == CONTROL` since `CONTROL` is not null.  You can just check `storyAdSegmentBranch != StoryAdSegmentExp.CONTROL`.

Reply:
Nvm